lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -191,8 +191,16 @@ apos.define('apostrophe-schemas', {
         field: field,
         type: type
       };
-      if (type === 'required') {
-        error.message = 'Required';
+      switch (type) {
+        case 'required':
+          error.message = 'Required';
+          break;
+        case 'min':
+          error.message = 'Min. permitted value is: ' + field.min;
```

Comment:
Great. The thinking was "red is better than nothing, we'll add in the messages later." About time. (: